<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>circuito</class>
 <widget class="QMainWindow" name="circuito">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>circuito</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>651</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Porta</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="porta_serial"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Velocidade</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="vel_serial"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="conectar">
           <property name="text">
            <string>Conectar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="status_conexao">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="desconectar">
           <property name="text">
            <string>Desconectar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Quadrante 1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Tempo aceso: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="t1"/>
             </item>
             <item>
              <widget class="QPushButton" name="inserir1">
               <property name="text">
                <string>Inserir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="q1">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="ligar1">
               <property name="text">
                <string>Ligar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="desligar1">
               <property name="text">
                <string>Desligar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Quadrante 2</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Tempo aceso:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="t2"/>
             </item>
             <item>
              <widget class="QPushButton" name="inserir2">
               <property name="text">
                <string>Inserir</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="q2">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="ligar2">
               <property name="text">
                <string>Ligar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="desligar2">
               <property name="text">
                <string>Desligar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
